### PR TITLE
Sponsor logo

### LINKED ISSUE #504: Add optional sponsor logo
Mocks TBD.

### PR DESCRIPTION
**Pull request by katiecrane**
- admins can add a sponsor logo
- database migration for new setting
### Issues:

Fixes #504

### CODE REVIEW

**Comment by DFurnes** [comment]
One more thing! :grimacing: Can you add a setting for a PNG fallback image and [set it to be used when SVGs aren't supported by the browser](https://github.com/DoSomething/voting-app/blob/dev/resources/views/partials/navigation.blade.php#L4)?

**Comment by katiecrane** [comment]
@DFurnes yaaasssss

File: database/seeds/BackgroundsTableSeeder.php [hunk 1]
```diff
@@ -20,5 +20,9 @@ public function run()
         $three = new Background();
         $three->saveImage(base_path('database/seeds/images/tabby3.jpg'));
         $three->save();
+
+        $four = new Background();
+        $four->saveImage(base_path('database/seeds/images/tongue-cat.png'));
```
Comment (DFurnes) [inline database/seeds/BackgroundsTableSeeder.php:25]: :tongue: :cat: !!!

**Comment by DFurnes** [comment]
Looks good to me! :ship: :cat: :tongue:

**Pull request merged**